workflow/scripts/extract-personal-data.py

```diff
@@ -1,3 +1,5 @@
+sys.stderr = open(snakemake.log[0], "w")
+
```

Comment:
Soll das wirklich über die import statements?

Reply:
This way expectations, that are throw when importing the other modules, are logged. Since this almost never happens, one could also include this line in the `if __name__ == __main__` block.

Reply:
btw, the `sys` module is provided via Snakemake.